### PR TITLE
Dynamo, Config, 0.3.5

### PR DESCRIPTION
**Pull request by ngr**

### CODE REVIEW

File: sosww/components/config.py [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+        self.config = {
+            'dynamo_client_config': {
+                'row_mapper':      {
+                    'env':          'S',
+                    'config_name':  'S',
+                    'config_value': 'S'
+                },
+                'required_fields': ['env', 'config_name', 'config_value'],
+                'table_name':      'config'  # If a table is not specified, this table will be used.
```
Comment (ngr) [inline sosww/components/config.py]: remove comment

Comment (nimirium) [inline sosww/components/config.py]: Remove comment

File: sosww/components/dynamo_db.py [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+        return result
+
+
+    @benchmark
+    def dict_to_dynamo(self, row_dict, add_prefix=None, strict=True):
+        """
+        Convert the row from regular dictionary to the ugly DynamoDB syntax. Takes settings from row_mapper.
+
+        e.g.                {'key1': 'value1', 'key2': 'value2'}
+        will convert to:    {'key1': {'Type1': 'value2'}, 'key2': {'Type2': 'value2'}}
```
Comment (nimirium) [inline sosww/components/dynamo_db.py]: {'Type1': 'value1'}

File: sosww/components/test/test_dynamo_db.py [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+                        ':range_col': {'N': str(row['range_col'])}
+                    }
+                )
+
+        items = result['Items']
+
+        self.assertTrue(len(items) > 0)
+
+    def test_dict_to_dynamo_strict(self):
+        dict_row = {'lambda_name': 'test_name', 'invocation_id': 'test_id', 'en_time': 123456}
```
Comment (nimirium) [inline sosww/components/test/test_dynamo_db.py]: hmmm...

**Pull request merged**